.github/workflows/reviewers.yml

```diff
@@ -0,0 +1,39 @@
+reviewers:
+  # The default reviewers
+  defaults:
+    - LysandreJik
+    - osanseviero
```

Comment:
Just for clarification, default reviewers are only assigned if no other reviewers are matched. Is that the expectation?

Reply:
That's not, so we can turn defaults into a group, and add that group to each PR. (Something we *couldn't* do well w/o the action)